api/storage_service.cc

```diff
@@ -1386,7 +1386,7 @@ void set_snapshot(http_context& ctx, routes& r, sharded<db::snapshot_ctl>& snap_
         if (!req_param<bool>(*req, "disable_snapshot", false)) {
             auto tag = format("pre-scrub-{:d}", db_clock::now().time_since_epoch().count());
             f = parallel_for_each(column_families, [&snap_ctl, keyspace, tag](sstring cf) {
```

Comment:
I don't understand the commit message. Are we preventing snapshots today? How are user errors involved? What are pedagogical reasons?

Reply:
reworded